src/base-apis.js

```diff
@@ -338,16 +338,28 @@ MatrixBaseApis.prototype.logout = function(callback) {
  * it is up to the caller to either reset or destroy the MatrixClient after
  * this method succeeds.
  * @param {object} auth Optional. Auth data to supply for User-Interactive auth.
+ * @param {boolean} erase Optional. If true, send erase=true query parameter which
+ * indicates that the account should be erased.
  * @param {module:client.callback} callback Optional.
  * @return {module:client.Promise} Resolves: On success, the empty object
  */
-MatrixBaseApis.prototype.deactivateAccount = function(auth, callback) {
-    let body = {};
-    if (auth) {
-        body = {
-            auth: auth,
-        };
+MatrixBaseApis.prototype.deactivateAccount = function(auth, erase, callback) {
+    if (typeof(erase) === 'function') {
```

Comment:
elsewhere we have a comment along the lines of "erase used to be callback" or words to that effect